src/common/jwt.service.js

```diff
@@ -1,15 +1,15 @@
 const ID_TOKEN_KEY = "id_token";
 
-export default {
-  getToken() {
-    return window.localStorage.getItem(ID_TOKEN_KEY);
-  },
-
-  saveToken(token) {
-    window.localStorage.setItem(ID_TOKEN_KEY, token);
-  },
-
-  destroyToken() {
-    window.localStorage.removeItem(ID_TOKEN_KEY);
-  }
+const getToken = () => {
```

Comment:
We would still not be able to import the function as named parameter.

Could you export `getToken` itself too?

It should look similar to:

```js
export const getToken = () => {...}

export default { getToken, ... }
```

You can see an example here on what it should look like: https://github.com/gothinkster/vue-realworld-example-app/blob/master/src/common/config.js

Reply:
oh, forgot that, I'll update